wikimedia/wikipedia-ios
# Settings swipe

**Phabricator:** https://phabricator.wikimedia.org/T217937

### Notes
* This PR adds the swipe to pop gesture on Settings' navigation stack, adding support for the swipe gesture form left to right on some view controllers pushed from this navigation controller.

### Test Steps
1. Launch the app, tap the Settings icon 
2. Tap on some option that is pushed from the controller (Search, Account/Login, Explore Feed and Notifications are eligible - on screens with the X close button on the top right this gesture shouldn't work).
3. Use the swipe from left to right gesture to return to the main Settings page.
4. Try to swipe back on the Settings page, nothing should happen here.
5. With the Settings page open, close the app and quit it on the background. Launch the App again (it should open on the Settings page, and repeat steps 2 - 4.
6. Disable the explore feed (it should be replaced by the Settings page), repeat steps 2 - 5.
